README.md

```diff
@@ -62,14 +62,14 @@ Create a new Volto project by using the `@plone/generator-volto` utility.
 
 It will bootstrap a Volto project in a folder of your choice with all the required
 boilerplate to start customizing your Volto site.
-
-    $ npm install -g yo @plone/generator-volto
-    $ yo @plone/volto
-
+```
+sudo npm install -g yo @plone/generator-volto
+yo @plone/volto
+```
 follow the prompts questions, provide `myvoltoproject` as project name then, when it finishes:
-
-    $ cd myvoltoproject
-
+```
```

Comment:
```suggestion
```shell
```

Reply:
All this pain can be avoided by the use of NVM, which installs a "local" personalized copy of the nodejs environment. This is documented in the [Volto Docs](https://docs.voltocms.com/getting-started/install/#install-nvm-nodejs-version-manager). Maybe this readme should just point to the docs.

Reply:
@tiberiuichim great minds think alike. https://github.com/plone/volto/pull/3131#issuecomment-1063315610

Reply:
This sounds to be a good suggestion, should I go on deleting all of these lines and just adding the link to the docs to it?

Reply:
I defer to the maintainers to make that call.